commands/team.go

```diff
@@ -314,8 +315,12 @@ func searchTeamCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 
 	sortedTeams := removeDuplicatesAndSortTeams(teams)
 
-	for _, team := range sortedTeams {
-		printer.PrintT("{{.Name}}: {{.DisplayName}} ({{.Id}})", team)
+	if len(sortedTeams) > 0 {
```

Comment:
I would suggest to make this similar to `searchUserCmdF`. Move the check to the loop above. And before appending, check if len(foundteams) is 0, then print - `"Unable to find team '"+ searchTerm "'"`